.github/workflows/test-pull-request-java-akka.yml

```diff
@@ -0,0 +1,29 @@
+name: Test Pull Request (Akka)
+
+on:
+  merge_group:
+  pull_request:
+    branches:
+      - main
+    paths:
+      - java/akka/**
+
+jobs:
+  smoke:
+    name: Smoke Tests
+    runs-on: ubuntu-latest
+    steps:
+    - name: Setup Go
+      uses: actions/setup-go@v4
+      with:
+        go-version: 1.18.x
+
+    - name: Checkout
+      uses: actions/checkout@v4
+
+    - name: Test Java Samples
+      run: |
+        ./scripts/smoke.sh --suite java/akka/smoke_test \
+                           --builder paketobuildpacks/builder:full \
```

Comment:
Let's drop full. The value we're going to get out of this is questionable, IMHO. Most Java users are on tiny or base.

Reply:
fine by me!